tests/Fixture/ArticlesFixture.php

```diff
@@ -13,9 +13,10 @@ class ArticlesFixture extends TestFixture
      */
     public $fields = [
         'id' => ['type' => 'string', 'null' => false, 'length' => 36],
-        'title' => ['type' => 'string', 'null' => false, 'default' => null],
         'created' => ['type' => 'datetime', 'null' => true, 'default' => null],
         'modified' => ['type' => 'datetime', 'null' => true, 'default' => null],
+        'title' => ['type' => 'string', 'null' => false, 'default' => null],
```

Comment:
Cause my brain imploded when seeing the un-cakeish order of fields ;p
I can undo that.